apps/src/templates/progress/ProgressButton.jsx

```diff
@@ -0,0 +1,136 @@
+/**
+ * New style button. Eventually we'd like to have this used more broadly (at which
+ * point it should be renamed to something more generic)
+ * This particular button is designed to operate in contexts where we have a solid
+ * background. When we're a button on top of an image, we may want something different.
+ */
+
+import React, { PropTypes } from 'react';
+import Radium from 'radium';
+import color from "@cdo/apps/util/color";
+import FontAwesome from '@cdo/apps/templates/FontAwesome';
+
+const ButtonColor = {
+  orange: 'orange',
+  gray: 'gray',
+  blue: 'blue',
+  white: 'white'
+};
+
+const ButtonSize = {
+  default: 'default',
+  large: 'large'
+};
+
+const styles = {
+  main: {
+    display: 'inline-block',
+    fontSize: 12,
+    fontFamily: '"Gotham 4r", sans-serif',
+    borderStyle: 'solid',
+    borderWidth: 1,
+    borderRadius: 3,
+    textDecoration: 'none',
```

Comment:
I actually fixed that hover by adding a cursor to the :hover after taking the screencast, but didn't bother to update the screencast :)